Nb final edits overall (#25)
R4EPI/R4EPIs-website: content/surveys/_index.en.md

![MSF Epidemiologist carrying out survey](images/msfepisurvey.jpg?width=30pc)

*Above: Suzanne Moja, 36, from Madrid, Spain is an epidemologist and supervises the team of MSF researchers in Bokoro, Chad.* ***Source: Charlotte Morris/MSF.*** 

**The survey templates have been developed to match the data that would be collected using 
[MSF Ethical Review Board pre-approved surveys](https://fieldresearch.msf.org/handle/10144/618942) on:** 

* ***Retrospective mortality and access to care***,  
* ***Vaccination coverage*** and   
* ***Malnutrition***.   

## Getting started

> The survey report templates are special R scripts called "R-Markdowns", and are included with **sitrep** package installation. 

Reviewer: ```suggestion
The survey report templates are special "RMarkdown" scripts, and are included with **sitrep** package installation. 
```

> RMarkdown scripts execute data cleaning and analytical R commands AND are themselves the outline of an automatically-produced report.

Reviewer: ```suggestion
RMarkdown scripts give you a way of writing your analysis code and report in one document by allowing you to interleave chunks of code that the computer will run in between explanatory text meant for a person to read.
```

**For assistance using the survey templates** 

Read more about RMarkdown scripts and how to modify the templates in the [Training Materials](https://r4epis.netlify.com/training/) pages of this website. 

The [case study walk-though](https://r4epis.netlify.com/training/walk-through/) offers you a step-by-step guide on how to use the templates to fit your data and run the analysis. 
Althrough the walk-through uses an outbreak template, the structure and techniques used to modify the templates are similar. 
At the start of the walk-through you can download the case study's data files and final template to follow along. 
In the walk-through, you will learn lessons for modifying the templates to fit your needs.

**To access the survey templates:**

> * Open Rstudio

Reviewer: ```suggestion
* Open RStudio
```

* Under the 'File' button (top left of your screen), click on the "+" 
* Then click on "R markdown...", 
* and select "From template"; 
you should be able to see all the options of templates that are available. For example "Mortality Survey". 

> ![example of opening and saving cholera template](images/opening_template.gif)

Reviewer: Make sure to give the section header room to breathe

```suggestion
![example of opening and saving cholera template](images/opening_template.gif)

```

{{% notice info %}}
The sampling stratgey **MUST** be considered by the epidemiologist using the templates in order to ensure an appropriate weighting is applied to the analysis.

> {{% /notice %}}

Reviewer: good catch!
